### DISCUSSION TITLE
mounts page

### DISCUSSION THREAD

**Discussion by bonesoul** [wowtrack/feedback#21] [Ideas]
create a mounts page

**Comment by Sahasrahla** [comment]
Not sure what you envision for this, but I think it would be neato to have a page like your Balance of Power checklist, but for the Legion Class Mounts. A quick check of the account's mount list will tell you which classes are already completed, and quest IDs can tell how far along the rest of them are.

**Reply by bonesoul** [reply]
Hi there,

I'm planning a similar page where you'll be able to filter mounts based on expansion and categories (Legion/Class mounts).
